packages/world/package.json

```diff
@@ -6,21 +6,26 @@
     "./worlds.json": "./worlds.json",
     "./mud.config": "./dist/mud.config.js",
     "./out/*": "./dist/out/*.js",
-    "./tree-lib": "./dist/tree-lib.js"
+    "./ts/*": "./dist/ts/*.js"
   },
   "typesVersions": {
     "*": {
       "mud.config": ["./dist/mud.config.d.ts"],
-      "out/*": ["./dist/out/*.d.ts"],
-      "tree-lib": ["./dist/tree-lib.d.ts"]
+      "ts/*": ["./dist/ts/*.d.ts"],
+      "out/*": ["./dist/out/*.d.ts"]
     }
   },
-  "files": ["src", "out", "dist", "worlds.json"],
+  "files": ["src", "out", "dist", "ts", "worlds.json"],
   "scripts": {
-    "build": "pnpm run build:mud && pnpm run build:js",
+    "build": "pnpm run codegen && pnpm run build:mud && pnpm run build:js",
     "build:js": "tsc",
     "build:mud": "mud build && abi-ts --extension .ts",
     "clean": "forge clean && shx rm -rf src/**/codegen",
+    "codegen": "pnpm run codegen:object-type && pnpm run codegen:init-objects && pnpm run codegen:init-recipes && pnpm run codegen:tree-lib && forge fmt",
```

Comment:
nit: could we codegen just one file that does all of these?

Reply:
you mean having a bash script for this? It is kind of the same as just doing it here no?

Reply:
yea cuz its easier to read than a bunch of chained `&&`, but not a strong preference, can keep what you have